core/modules/catalog-next/store/category/actions.ts

```diff
@@ -176,7 +177,7 @@ const actions: ActionTree<CategoryState, RootState> = {
     let resultFilters = aggregationFilters
     const filtersKeys = Object.keys(filters)
     if (categoryMappedFilters && filtersKeys.length) {
-      resultFilters = Object.assign({}, categoryMappedFilters, omit(aggregationFilters, filtersKeys))
+      resultFilters = Object.assign(cloneDeep(categoryMappedFilters), cloneDeep(omit(aggregationFilters, filtersKeys)))
```

Comment:
```suggestion
      resultFilters = cloneDeep(Object.assign(categoryMappedFilters, omit(aggregationFilters, filtersKeys)))
```

Reply:
In this case too, we need either multiple clone operations or Object.assign({}, ...)